README.md

```diff
@@ -174,6 +174,42 @@ Run
 Check the downloads folder in `test-tree/downloads` and ensure that the `downloads/nested` directory has a structure like the nested directory uploaded in the [nested uploads test](#nested-uploads).
 
 
+#### Quantity Tests
+
+To test uploads/downloads for a large number of files we need to "a large number" of files on either side of the process.
+
+To generate a number of files with a specific size in `test-tree/special-files`, run `./create-files.py --quantity 10 --size 10000 --root-name "10-10KB"`
```

Comment:
To generate a number of files with a specific size .. run ... . *In this case, the command would generate 10, 10 bytes files.*